tests/standardise/test_standardise.py

```diff
@@ -309,6 +311,35 @@ def test_standardise_footprint():
     assert "tmb_europe_test_model_10m" in results
 
 
+def test_standardise_align_footprint():
```

Comment:
Could you add a short docstring here describing what the test is doing?

Reply:
Done in a recent commit